packages/ui/src/common/email-or-phone-number-field.tsx

```diff
@@ -32,8 +34,7 @@ export function EmailOrPhoneNumberField({
       onChange={setShowPhoneNumberField}
       className={linkButton({ size: 'sm', disabled: props.disabled })}
     >
-      <span className='sr-only'>{toggleDescription}</span>
```

Comment:
Removed the hidden text to make sure the button is selectable via the visible text label.